app/modules/utils.py

```diff
@@ -35,7 +35,7 @@ def is_valid_sex(value):
 
 
 def is_valid_latitude(value):
-    if not value or not isinstance(value, float):
+    if not isinstance(value, float) and not isinstance(value, int):
```

Comment:
why is this necessary ? The previous fix already cast ints that should have been floats into floats which is what the asset group metadata validation does too. 
How else is this called when the lat/long is not a float?